upgrade/upd-2.5.4-to-2.5.5/index.php

```
@@ -83,39 +89,52 @@ public function apply_keys()
             return false;
         }
 
         return true;
     }
 
-    public function __construct()
+    /**
+     * @param XoopsMySQLDatabase $db      database connection
+     * @param UpgradeControl     $control upgrade control instance
+     */
+    public function __construct(XoopsMySQLDatabase $db, UpgradeControl $control)
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)